src/AWS/CloudWatchLogs/CloudwatchLogs.js

```diff
@@ -0,0 +1,9 @@
+"use strict"
+
+var AWS = require('aws-sdk')
+
+exports.makeClientImpl = (params) =>
+  () => new AWS.CloudWatchLogs(params)
```

Comment:
is cloud watch logs actually a different service in sdk terms? Otherwise I would just move it to `CloudWatch`

Reply:
Ok just saw that indeed it is, so we can leave it like this 👍

Reply:
Ok will click resolve button here :)